src/main/java/Eln.java

```diff
@@ -0,0 +1,47 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    static String LINE = "____________________________________________________________";
```

Comment:
Maybe consider making this private (if it wont be used out of this class) and final (constant)?
```suggestion
    private static final String LINE = 
    "____________________________________________________________";
```